tests/data/experiments/lr-predict-illegal-transformations/existing_experiment/output/lr_postprocessing_params.csv

```diff
@@ -1,2 +1,2 @@
 h1_mean,h1_sd,train_predictions_mean,train_predictions_sd,trim_max,trim_min
-3.42,0.8154323146156515,3.420000000000002,0.5799500693653682,6.0,1.0
```

Comment:
Oh, you changed the parameters of the existing experiment, right?

Reply:
Yeah, the file changed values because of some changes in pandas.